### ISSUE TITLE
enable enhanced monitoring?

### ISSUE THREAD

**Issue by rexroof** [cloudposse/terraform-aws-rds-cluster#20]
Can this module enable enhanced monitoring on a new aurora cluster?

**Comment by osterman** [comment]
I don't believe we've added support for that yet. It would be a welcome enhancement that others would likely want too. Contributions welcome, but it's not currently on our list.
